zip-1014.rst

```diff
@@ -297,22 +310,23 @@ conditions, and the prescribed use of funds, such that substantial violation,
 not promptly remedied, will permit the other party to issue a modified version
 of Zcash node software that removes the violating party's Dev Fund slice, and
 use the Zcash trademark for this modified version. The slice's funds will be
-reassigned to ZF-MG (whose integrity is legally protected by the Restricted
+reassigned to MG (whose integrity is legally protected by the Restricted
 Fund treatment).
 
 
 Future Community Governance
 ---------------------------
 
 Decentralized community governance is used in this proposal via the Community
-Panel as input into the Major Grant Review Committee which governs the `ZF-MG
-slice (Zcash Foundation, for major grants)`_.
+Panel as input into the Major Grant Review Committee which governs the
+`MG slice (Major Grants)`_.
 
 It is highly desirable to develop robust means of decentralized community
-voting and governance --- either by expanding the Community Panel
-or a successor mechanism --- and to integrate them into both of these
-processes, by the end of 2021. ECC and ZF SHOULD place high priority on such
-development and its deployment, in their activities and grant selection.
+voting and governance –either by expanding the Community Panel or a
+successor mechanism– and to integrate them into this and any other
+pertinent governance processes, by the end of 2021. ECC and ZF SHOULD place
```

Comment:
Can this be changed to `and to integrate them into this process by the end of 2021`? Basically remove "and any other pertinent governance processes." We shouldn't make that edit at this stage given our attempts to minimize the number of changes to ZIP 1014, and the community can decide through future ZIPs whether this process should be used for other pertinent governance decisions.

Reply:
ACK.

Reply:
ACK.